README.md

```diff
@@ -105,28 +105,37 @@ pip install -e .
 
 ### Run an experiment
 
-There are two ways to run an experiment.
+There are two ways to run an experiment of multi-partner learning approaches and/or contributivity methods on a given dataset.
```

Comment:
I would remove "on a given dataset" since you can run a experiment with multiple scenario, so on multitple datasets

Reply:
Done